.circleci/config.yml

```diff
@@ -56,6 +58,8 @@ jobs:
             pylint tests/unit tests/integration/devtools tests/integration/renderer tests/integration/dash_assets -d all -e C0410,C0411,C0412,C0413,W0109
             cd dash-renderer && npm ci
             npm run lint:test && npm run format:test
+            cd ..
+            cat *.md | aspell --lang=en --encoding=utf-8 --personal=./.aspell.en.pws list | sort -u && exit 0
```

Comment:
Thoughts:

1. Structure the output / group mistakes per file to make it easier to update/fix issues

e.g. I'm sure this can be cleaned up:
```
for f in $(find . -name "*.md" -not -path "**/node_modules/*"); do echo -e $f "\n" $(cat $f | aspell list --lang=en --encoding=utf-8 --personal=./.aspell.en.pws | sort -u) "\n"; done
```
outputs:
```
./CODE_OF_CONDUCT.md

./.pytest_cache/README.md
 lf pytest's

./CHANGELOG.md

./README.md

./CONTRIBUTING.md

./.github/pull_request_template.md
 TODO github optionals

./.github/ISSUE_TEMPLATE/feature_request.md
 assignees

./.github/ISSUE_TEMPLATE/bug_report.md
 assignees
```

2. How do we go about changing `exit 0` for something that will actually trigger a failure? There's no point in adding a check if we are not going to enforce it.

3. Fix remaining items